app/models/resources.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-len */
```

Comment:
Should we split that into two directories then to save confusion:

e.g. 
\resource\data\articles.js
\resource\data\categories.js

\resource\model\dashboard-view-model.js